doto-backend/src/config/token-setup.js

```diff
@@ -1,17 +1,18 @@
 const jwt = require('jsonwebtoken')
 
-function generateAccessToken(user){
+function generateAccessToken(user) {
     return jwt.sign(user, process.env.ACCESS_TOKEN_SECRET)
 }
 
-function authenticateToken(req, res, next){
+function authenticateToken(req, res, next) {
     const authHeader = req.headers['authorization']
     const token = authHeader && authHeader.split(' ')[1]
 
-    if(token == null) return res.sendStatus(401)
+    if (token == null) { return res.sendStatus(401) }
+
+    jwt.verify(token, process.env.ACCESS_TOKEN_SECRET, (err, user) => {
```

Comment:
It might be better to store all environment variables in an object at the start as it makes it less cluttered.